cleanupBucket.py

```diff
@@ -114,8 +114,6 @@ def setupConfig(config):
     return config
 
 
-
-
 def cleanupBucket(s3_client, bucket):
```

Comment:
This is a next level suggestion, but I know the s3 api only allows 1000 delete requests per api call.

To speed this up, I suggest using an async library such as aiobotocore so you can make these requests async and not synchronously. It will speed it up many factors but at the same time, it's a more difficult programming paradigm. 

https://github.com/aio-libs/aiobotocore

Reply:
definitely a good point for a future project